src/CloudService/CloudService.Autorest/docs/Invoke-AzCloudServiceRebuild.md

```diff
@@ -15,17 +15,31 @@ If you do not want to initialize storage resources, you can use Reimage Role Ins
 
 ### RebuildExpanded (Default)
 ```
-Invoke-AzCloudServiceRebuild -CloudServiceName <String> -ResourceGroupName <String> -RoleInstance <String[]>
- [-SubscriptionId <String>] [-DefaultProfile <PSObject>] [-AsJob] [-NoWait] [-PassThru] [-Confirm] [-WhatIf]
+Invoke-AzCloudServiceRebuild -CloudServiceName <String> -ResourceGroupName <String> [-SubscriptionId <String>]
+ [-RoleInstance <String[]>] [-DefaultProfile <PSObject>] [-AsJob] [-NoWait] [-PassThru] [-Confirm] [-WhatIf]
```

Comment:
roleinstance is path parameter which should be mandatory: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/Microsoft.Compute/CloudserviceRP/stable/2022-09-04/cloudService.json#L407